src/Providers/Storage/Illuminate.php

```diff
@@ -11,13 +11,13 @@
 
 namespace Tymon\JWTAuth\Providers\Storage;
 
-use Illuminate\Cache\CacheManager;
 use Tymon\JWTAuth\Contracts\Providers\Storage;
+use Illuminate\Contracts\Cache\Store as StoreContract;
```

Comment:
Did you tested it? Laravel cannot instatiate Cache\Store contract.
CacheManager is an implementation of Cache\Factory contract, so it class should be use instead of Cache\Store.

In my opinion laravel Managers are so flexible, so there's no need to use contracts in this case (jwt packages allows custom cache/auth provider classes).